### DISCUSSION TITLE
Ask a question about UI translation.

### DISCUSSION THREAD

**Discussion by kqbi** [apache/skywalking#12518] [Q&A]
Previously, I deployed version 9.0.0, and then there were many tabs on the left side of the UI. However, after deploying version 9.7.0or10.0.0, only a few tabs appear on the left. What could be the reason for this? How should I configure it?
I've just started using SkyWalking and there's a lot I don't understand. It might be a minor issue, but I would greatly appreciate everyone's help in looking into it! Thank you so much!!!
![9 7](https://github.com/user-attachments/assets/07e3fcce-247c-4d97-afbe-e59a7b2b457a)

**Comment by kqbi** [comment]
Using the default configuration.

**Comment by wu-sheng** [comment]
Marketplace list all. It shows up when you have relative services detected.
